### ISSUE TITLE
Crashed on removing weapon

### ISSUE THREAD

**Issue by oylsister** [oylsister/ZombieSharp#4]
Server will crash when after MotherZombieInfect excuted, after test several time, This is not occurred when removing weapon function is not active. Strange enough when Infect with Command it seem to work fine. however it still crash when is excuted from other function.

```
WriteEnterPVS: GetEntServerClass failed for ent 242.
0 WriteEnterPVS: GetEntServerClass failed for ent 242.
```

**Comment by oylsister** [comment]
We still need to find a method to prevent zombie picking up the weapon.

**Comment by oylsister** [comment]
Fixed on #5
